monitoring/uss_qualifier/scd/simulator/utils.py

```diff
@@ -0,0 +1,104 @@
+from pathlib import Path
+from typing import List, Union, Literal
+from geojson import Feature
+import enum
+from shapely.geometry import LineString, Polygon
+from monitoring.monitorlib.typing import ImplicitDict, StringBasedDateTime
+from monitoring.monitorlib.scd import Altitude, Volume4D,  OperationalIntentState
+from pathlib import Path
+
+class OutputSubDirectories(ImplicitDict):
```

Comment:
Let's hold off on defining data structures for the simulator in this PR as they will require more discussion -- the data structures mirror the ASTM data types and the automated testing API data types seem pretty close to being done for this PR.

Reply:
OK removing this.